src/main/java/com/rackspace/jenkins_nodepool/NodePool.java

```diff
@@ -145,6 +143,11 @@ public static CuratorFramework createZKConnection(String connectionString,
      */
     private String requestor;
 
+    /**
+     * Timeout for a node request
+     */
+    private String requestTimeout;
```

Comment:
Store request timeout as String or Integer?  I guess it's a matter of style - convert the value earlier or later?

Reply:
Done